### PR TITLE
First implementation of the 2-prong secondary vertex reconstruction

### PR DESCRIPTION
**Pull request by ginnocen**

### CODE REVIEW

**Comment by ktf** [comment]
@ginnocen I had to fix the executable name.

**Comment by ginnocen** [comment]
Thanks @ktf

**Comment by aphecetche** [comment]
btw, is it understood how it's possible that this PR's error has been polluting other PRs ?

**Comment by ktf** [comment]
Because CMake does not delete removed entries, and since we use incremental builds to save time,  the leftover is there. Time to move to bazel.

File: Framework/AnalysisTutorial/src/vertexinghf/vertexerhf.cxx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        //LOGF(info, "track2 %f", trackparvar2.getSigmaY2());
+
+        df.setUseAbsDCA(true);
+        int nCand = df.process(trackparvar1, trackparvar2);
+        printf("\n\nTesting with abs DCA minimization: %d candidates found\n", nCand);
+        // we can have up to 2 candidates
+        for (int ic = 0; ic < nCand; ic++) {
+          const o2::base::DCAFitter::Triplet& vtx = df.getPCACandidate(ic);
+          LOGF(info, "print %f", vtx.x);
+          //LOGF(info, "LABELELELELELLE %d", track2.index());
```
Comment (ktf) [inline Framework/AnalysisTutorial/src/vertexinghf/vertexerhf.cxx]: Please do not leave commented code around.

Comment (ginnocen) [reply Framework/AnalysisTutorial/src/vertexinghf/vertexerhf.cxx]: @ktf sorry for that. I removed them.

**Comment by ktf** [comment]
Looks good to me. Is it ok if I squash it?

**Comment by ginnocen** [comment]
Sure please go ahead and squash it 
Thanks a lot

**Comment by ktf** [comment]
I rebased the PR on to of the current dev branch.

**Pull request closed without merging**